.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,56 @@
+name: 'Bug report'
+labels: ['bug', 'triage']
+description: Create a report to help Epinio to improve
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Current Behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. In this environment...
+      2. With this config...
+      3. Run '...'
+      4. See error...
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Environment
+    description: |
+      examples:
```

Comment:
We're also very interested in the cluster provider. I guess that's true for the helm-charts repo, too. See https://github.com/epinio/epinio/blob/main/.github/ISSUE_TEMPLATE/bug_report.md